.ci/common.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# Attention, there is no "-x" to avoid problems on Travis
+set -e
+
+# result of command goes to $RUN_JOB
+# 0 = not run (skip)
+# 1 = run (no skip)
+function should_run_job {
+    SKIP_JOB_BY_FILES=$1
+    SKIP_FILES=$2
+    DEBUG=$3
```

Comment:
even in java extending debug is done by some "global" variables, lets remove this argument from function and rely on global variable "DEBUG".

Reply:
The main purpose of DEBUG as a parameter was to not have it on for every CI step.
I was going to enable it for a few, but this caused an issue when a step without debug didn't behave as expected.

Reply:
To enable debug, you will need change code. Instead of change in arguments, you can simply define this variable in CI job config, or to do for all - in function.

We are not over verbose, so you can always print this info, no that much need for extra log flag

Reply:
Done.